doc/user/content/sql/tail.md

```diff
@@ -1,48 +1,118 @@
 ---
 title: "TAIL"
-description: "`TAIL` continually reports updates that occur to a source or view."
+description: "`TAIL` streams updates from a relation as they occur."
 menu:
-    main:
-        parent: "sql"
+  main:
+    parent: "sql"
 ---
 
-`TAIL` continually reports updates that occur to a source or view.
-For materialized sources or views this data only represents updates that occur after running the `TAIL` command.
-For non-materialized sources or views, all updates are presented.
+`TAIL` streams updates from a source, table, or view as they occur.
 
-Tail will continue to run until cancelled, or until all updates the tailed item could undergo have been presented. The latter case may happen with static views (e.g. `SELECT true`), files without the `tail = true` modifier, or other settings in which a collection can cease to experience updates.
+## Conceptual framework
 
-In order for a tail to be possible, all involved sources must be valid to read from at the chosen timestamp.
-A tail at a given timestamp might not be possible if data it would report has already been compacted by Materialize.
+The `TAIL` statement is a more general form of a [`SELECT`](/sql/select)
+statement. While a `SELECT` statement computes a relation at a moment in time, a
+tail operation computes how a relation *changes* over time.
+
+Fundamentally, `TAIL` produces a sequence of updates. An update describes either
+the insertion or deletion of a row to the relation at a specific time. Taken
+together, the updates describes the complete set of changes to a relation, in
+order, while the `TAIL` is active.
+
+Clients can use `TAIL` to:
+
+  - Power event processors that react to every change to a relation.
+  - Replicate the complete history of a relation while the `TAIL` is active.
 
 ## Syntax
 
 {{< diagram "tail-stmt.svg" >}}
 
 Field | Use
 ------|-----
-_object&lowbar;name_ | The item you want to tail
-_timestamp&lowbar;expression_ | The logical time to tail from onwards (either a number of milliseconds since the Unix epoch, or a `TIMESTAMP` or `TIMESTAMPTZ`).
+_object&lowbar;name_ | The name of the source, table, or view that you want to tail.
+_timestamp&lowbar;expression_ | The logical time to tail from onwards, either as a [`bigint`] representing milliseconds since the Unix epoch, or as a [`timestamp`] or [`timestamptz`].
```

Comment:
Since this is no longer output as a text string, this can no longer be a timestamp and is always a bigint now.

Reply:
Thanks, done.